### DISCUSSION TITLE
Project not evolving much

### DISCUSSION THREAD

**Discussion by ewjoachim** [procrastinate-org/procrastinate#454] [General]
Just a small discussion to let people know why this project is moving less these days.

This open-source project was born in a corporate context at a time where a few people had reasonably enough time to contribute & maintain. We tend to have a little less time during our work hours now. I'd be glad to continue contributing to it on my own time regularly, but because this project is under `peopledoc`, this means any PeopleDoc employee can only contribute using their corporate computer, and I prefer not to touch my corporate computer outside of working hours.

Alternatives are:
- I create a second GitHub account to contribute, which I'd rather not
- We move this project to its own org, `procrastinate/procrastinate`
- Or I contribute only during my work hours, which might make is much less often.

I'll try to push for option B.

**Comment by elemoine** [comment]
I'd be a big +1 for moving the project to its own org. This project has been open-source from day 1, it's of general interest, and it's always been built and developed with the community in mind. For the reasons, to me it makes sense to move it to its own org.

**Comment by ewjoachim** [comment]
Closed by https://github.com/procrastinate-org/procrastinate/pull/459 :D
